### PR TITLE
sets: Update theory reference and smt2 examples.

### PR DESCRIPTION
**Pull request by aniemetz**

### CODE REVIEW

File: docs/theories/sets-and-relations.rst [hunk 1]
```diff
@@ -115,7 +115,7 @@ The set complement operator for ``(Set T)`` is interpreted relative to the
 interpretation of the universe set for ``(Set T)``, and not relative to the set
 of all elements of sort ``T``.
 That is, for all sets ``X`` of sort ``(Set T)``, the complement operator is
-such that ``(= (complement X) (setminus (as univset (Set T)) X))``
+such that ``(= (set.complement X) (set.minus (as univset (Set T)) X))``
```
Comment (ajreynol) [inline docs/theories/sets-and-relations.rst:118]:
```suggestion
such that ``(= (set.complement X) (set.minus (as set.universe (Set T)) X))``
```

File: docs/theories/sets-and-relations.rst [hunk 2]
```diff
@@ -189,11 +189,11 @@ More details can be found in :cite:`MengRTB17`.
 |                      |                                              |                                                                                    |
 |                      |                                              | ``Term X = solver.mkConst(s, "X");``                                               |
 +----------------------+----------------------------------------------+------------------------------------------------------------------------------------+
-| Transpose            | ``(transpose X)``                            | ``Term t = solver.mkTerm(Kind::TRANSPOSE, X);``                                    |
+| Transpose            | ``(rel.transpose X)``                        | ``Term t = solver.mkTerm(Kind::REL_TRANSPOSE, X);``                                |
```
Comment (ajreynol) [inline docs/theories/sets-and-relations.rst:192]: I think we renamed to `RELATION_` not `REL_`.

**Review by ajreynol — CHANGES_REQUESTED** [review]

File: docs/theories/sets-and-relations.rst (hunk 2 above)
Comment (aniemetz) [reply docs/theories/sets-and-relations.rst:192]: Right. Fixed.

**Review by mudathirmahgoub — APPROVED** [review]
LGTM

**Review by ajreynol — APPROVED** [review]

**Pull request merged**